src/components/FollowPlaylist/FollowPlaylist.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ChangeEvent, FormEvent, useState } from "react";
+import { useDispatch } from "react-redux";
+import { useHistory } from "react-router-dom";
+import { playlistsAsyncActions } from "../../store/slices/playlists/slice";
+import { Button, Container, FormWrapper, HelperText, StyledInput, Title, Wrapper } from "./styles";
+
+const FollowPlaylist = () => {
+    const [id, setId] = useState<string>("");
+    const [helperText, setHelperText] = useState<string | null>(null);
+    const dispatch = useDispatch();
+    const history = useHistory();
+
+    const saveId = (event: ChangeEvent<HTMLInputElement>) => {
+        let id = event.target.value.trim();
+        if (id.length === 0) {
+            setHelperText("Cannot be empty!");
+            setId("");
+        } else if (id.length > 20) {
```

Comment:
why do we need this? it is not the form job to determinate if the lenght/value is right. we dont have to tell the user that this is too long. let them submit a value, and if that is not right, the backend will send an error

Reply:
They will enter 20 character long ids, where they cannot tell at once, if they have made a mistake or not. This frontend validation helps them to double check and correct the id as soon as possible, if there is a mistake. It is like an early return. Because why to send even to backend, if we know for sure that it cannot be good anyway? 

I would leave both frontend and backend validation for better user experience and preventing unnecessary backend traffic.

Reply:
i understand your point of view, but still i think this is an antipattern. what if we rewrite the invitation, for example instead of the roomId (which i think is a bad practice, but for this project is good enough) the app generates a key which can be used for joining? then we have to rewrite the code here aswell. i know its "nothing" but so does a backend call with the wrong data.

Reply:
I do not agree that making validation on both frontend and backend would be an antipattern, it is quite usual, as far as I see. Indeed, if we would change the logic for following a playlist, then we would need to change the code, as it is happens with many other changes that we might make to the app in the future. 

If the playlist code would be a simple 4 digit code, then it would not be a big deal for sure. But it is a 20-digit code, which the user does not even know, what it should be like, so they do not realise, if they make any mistake during the copy-pasting. It simply improves the user experience, if they receive a feedback right away if they made sg. wrong. 

For sure it would be a mistake to replace backend validation with frontend validation, but having them both does not have any serious drawbacks, as I see it.